app/graphql/mutations/create_task.rb

```diff
@@ -0,0 +1,18 @@
+module Mutations
+  class CreateTask < BaseMutation
+    argument :input, Types::Inputs::CreateTaskInput, required: true
+
+    type Types::Payloads::CreateTaskPayload
+
+    def resolve(input:)
+      project = Project.find(input[:project_id])
```

Comment:
This variant are shorter
```suggestion
      project = Project.find(input.project_id)
```

Reply:
this variant IS shorter:)

Reply:
lol, thank u ))